<launch>
  <node pkg="imu_driver" type="driver.py" name="imu_driver_node" output="screen">
    <param name="port" type="string" value="/dev/ttyUSB0" />
  </node>
</launch>

